README.md

```diff
@@ -63,7 +63,7 @@ Add the following code to the `<dependencies>` section of your project:
 Add the following line to the `dependencies` section of your `build.gradle`:
 
 ```groovy
-implementation 'com.meilisearch.sdk:meilisearch-java:0.6.0'
+implementation 'com.meilisearch.sdk:meilisearch-java:0.7.0'
```

Comment:
@alallema https://github.com/meilisearch/meilisearch-java/blob/main/.code-samples.meilisearch.yaml#L375 you need to change this file also, right? (I was looking at the contributing file)